### PR TITLE
isolate the private key so it is not exposed at the public jwks endpoint

### PR DESCRIPTION
**Pull request by nathanloyer**
#123

### CODE REVIEW

File: lib/onc_certification_g10_test_kit.rb [hunk 1]
```diff
@@ -74,11 +74,12 @@ class G10CertificationSuite < Inferno::TestSuite
     end
 
     def self.jwks_json
-      @jwks_json ||= File.read(File.join(__dir__, 'onc_certification_g10_test_kit', 'bulk_data_jwks.json'))
+      bulk_data_jwks = JSON.parse(File.read(File.join(__dir__, 'onc_certification_g10_test_kit', 'bulk_data_jwks.json')))
+      @jwks_json ||= { keys: bulk_data_jwks['keys'].select { |key| key['key_ops']&.include?('verify') } }.to_json
```
Comment (Jack-Fraser16) [inline lib/onc_certification_g10_test_kit.rb:78]:
```suggestion
      @jwks_json ||= JSON.pretty_generate({ keys: bulk_data_jwks['keys'].select { |key| key['key_ops']&.include?('verify') } })
```
Small readability fix for the `.well-known` endpoint

**Review by Jack-Fraser16 — CHANGES_REQUESTED** [review]
Looks great - thank you for this fix and for helping us carry over parts of our legacy code, we really appreciate it! I commented a readability change (JSON.pretty_generate), feel free to take it or leave it. I'm also going to ask you to run `rubocop -A` and commit the changes from that, then I'll approve this and we can get it in. Again, thanks for your contribution!

**Comment by nathanloyer** [comment]
@Jack-Fraser16 made those updates. On the rubocop part, I found that just running `rubocop -A` made the formatting not very good in my opinion, so I instead made some manual changes that rubocop seems to be content with. Let me know if you have any other requests.

**Review by Jack-Fraser16 — APPROVED** [review]

**Comment by nathanloyer** [comment]
Thanks @Jack-Fraser16

**Pull request merged**